arviz/tests/test_stats.py

```diff
@@ -262,9 +262,26 @@ def test_loo_warning(centered_eight):
 
 
 def test_psislw():
+    np.random.seed(8)
     linewidth = np.random.randn(20000, 10)
     _, khats = psislw(linewidth)
     assert_array_less(khats, 0.5)
+    assert khats.shape == (10,)
+    assert_almost_equal(
```

Comment:
this seems pretty brittle/hard to maintain - I might leave the random seed, but get rid of the hardcoded k-hats. @canyon289 any thoughts?

Reply:
alternatively, adding a comment explaining _what_ this testing and why it is here. i just imagine being confused seeing this list of numbers in a year or two!

Reply:
Instead of using random data, we could use eight schools for the test. Assert that the values of loo.pareto_k and psislw and equal. We would get rid of these harcoded indices as well.